APTITUDE/APTITUDETEST.Data/Data/TestRepository.cs

```diff
@@ -317,10 +317,10 @@ public async Task<JsonResult> AddTestQuestions(TestQuestionsVM addTestQuestion)
                 _context.Add(testQuestionsToBeAdded);
                 _context.SaveChanges();
 
-                TestQuestionsCount testQuestionsCountToBeAdded = new TestQuestionsCount();
 
                 foreach (var testQuestionCount in addTestQuestion.TestQuestionsCount)
                 {
+                    TestQuestionsCount testQuestionsCountToBeAdded = new TestQuestionsCount();
```

Comment:
replace this line with - testQuestionsCountToBeAdded = new TestQuestionsCount();